Source/WTF/Scripts/Preferences/UnifiedWebPreferences.yaml

```diff
@@ -1964,11 +1964,11 @@ DataDetectorTypes:
   condition: ENABLE(DATA_DETECTION)
   defaultValue:
     WebKitLegacy:
-      default: DataDetectorType::None
+      default: static_cast<DataDetectorType>(0)
```

Comment:
Would using `refinedType: OptionSet<WebCore::DataDetectorType>` above work instead?

Reply:
nope :(

```
/Users/richardrobinson/Build/Products/Debug-maccatalyst/usr/local/include/wtf/CrossThreadCopier.h:88:39: error: no member named 'isolatedCopy' in 'WTF::OptionSet<WebCore::DataDetectorType>'
        return std::forward<U>(value).isolatedCopy();
/Users/richardrobinson/Build/Products/Debug-maccatalyst/usr/local/include/wtf/CrossThreadCopier.h:246:55: note: in instantiation of function template specialization 'WTF::CrossThreadCopierBase<false, false, WTF::OptionSet<WebCore::DataDetectorType>>::copy<const WTF::OptionSet<WebCore::DataDetectorType> &>' requested here
    return CrossThreadCopier<std::remove_cvref_t<T>>::copy(std::forward<T>(source));
/Users/richardrobinson/Build/Products/Debug-maccatalyst/DerivedSources/WebCore/Settings.cpp:1923:9: note: in instantiation of function template specialization 'WTF::crossThreadCopy<const WTF::OptionSet<WebCore::DataDetectorType> &>' requested here
        crossThreadCopy(dataDetectorTypes),
1 error generated.
```

Reply:
The default value is an `OptionSet<DateDetectorType>` value, not a `DataDetectorType` value. I suggest trying `{ }`.